Portal/vip-portal/src/main/resources/vm/wrapper.vm

```diff
@@ -26,6 +26,11 @@ function checkBosh {
     then
       info "Installing Boutiques locally"
       pip install boutiques --user    
+      if [ $? != 0 ]
+      then
+        error "pip install boutiques failed, exiting"
+        return 1
+      fi
```

Comment:
you could just do something like `pip install boutiques || error "message"` and avoid the if statement.